src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
+  constructor (props) {
+    super(props);
+    const { percentage, showPercentageNum, className } = props;
+
+    this.percentage = percentage || 0;
+    this.showPercentageNum = showPercentageNum;
+    this.className = className;
+  }
+
+  getBarClass () {
+    return cx(
+      'progress-bar',
+      this.className);
+  }
+
+  render () {
+    const progressBarWidth = this.percentage > 100 ? 100 : this.percentage;
```

Comment:
I think we should throw and error if the percentage is greater than 100 since percentage cannot be more than 100

Reply:
Good idea. I can make this as a public util function like `numInRange(min, max)`. Then this function can be reused.